tests/unit/nupic/research/sp_learn_inference_test.py

```diff
@@ -138,6 +140,7 @@ def _runLearnInference(self,
       print "Test succeeded"
 
 
+  @unittest.skip("Currently fails due to switch from FDRCSpatial2 to SpatialPooler.")
```

Comment:
It is not possible use the new SP to perform this test due to it not have explicit methods to compute inference. So I skipped it. Now can remove it, try find alternatives, or then keep it as `skipped` until future review as addressed in https://github.com/numenta/nupic/issues/1895 .